.gitignore

```diff
@@ -1,5 +1,23 @@
+# Node
 node_modules
 .npmrc
 
+# JetBrains
+**/.idea/workspace.xml
+**/.idea/tasks.xml
+**/.idea/shelf
```

Comment:
PhpStorm config is already under git: https://github.com/AmazeeLabs/silverback-mono/blob/development/.idea/.gitignore

So I guess we don't need this.